### DISCUSSION TITLE
[PHP] can php agent support getting trace id?

### DISCUSSION THREAD

**Discussion by brucezo** [apache/skywalking#10630] [Q&A] [answered]
can php agent support getting trace id?

**Comment by jmjoy** [comment]
Not yet.

I think this information can be automatically injected by skywalking-agent in $_SERVER or $_ENV.

**Reply by brucezo** [reply]
Has this injection feature been supported?

**Reply by jmjoy** [reply, marked as answer]
Not yet.
